protocol/protocol.go

```diff
@@ -40,20 +40,26 @@ func (p *Manager) Stop() error {
 func newRouterInitializer(netAdapter *netadapter.NetAdapter, dag *blockdag.BlockDAG) netadapter.RouterInitializer {
 	return func() (*netadapter.Router, error) {
 		router := netadapter.NewRouter()
-		err := router.AddRoute([]string{wire.CmdTx}, startDummy(netAdapter, router, dag))
+		err := router.AddRoute([]string{wire.CmdPing, wire.CmdPong}, startPing(netAdapter, router, dag))
```

Comment:
Can you undo changes to avoid redundant conflicts?

Reply:
I kinda want the PingPong flow to be there so that we can test connections until the real ping-pong flow is implemented